src/main/java/io/github/thebusybiscuit/slimefun4/core/services/profiler/inspectors/PlayerPerformanceInspector.java

```diff
@@ -27,34 +28,62 @@ public class PlayerPerformanceInspector implements PerformanceInspector {
      */
     private final UUID uuid;
 
+    /**
+     * The order type of the timings.
+     */
+    private final SummaryOrderType orderType;
+
     /**
      * This creates a new {@link PlayerPerformanceInspector} for the given {@link Player}.
      * 
      * @param player
      *            The {@link Player}
+     * @param orderType
+     *            The {@link SummaryOrderType} of the timings
      */
-    public PlayerPerformanceInspector(@Nonnull Player player) {
+    public PlayerPerformanceInspector(@Nonnull Player player, @Nonnull SummaryOrderType orderType) {
```

Comment:
You can still use parameters are nonnull by default but other than that this looks good to me